src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -116,8 +117,8 @@ void listen_to_kafka_topic(KafkaConnector *kstream, Partitioner &graphPartitione
         obj["destination"] = destinationJson;
         long temp_s = partitionedEdge[0].second;
         long temp_d = partitionedEdge[1].second;
-        workerClients.at((int)partitionedEdge[0].second)->publish(sourceJson.dump());
-        workerClients.at((int)partitionedEdge[1].second)->publish(destinationJson.dump());
+        workerClients.at((int)partitionedEdge[0].second)->publish(obj.dump());
+        workerClients.at((int)partitionedEdge[1].second)->publish(obj.dump());
```

Comment:
```suggestion
        string objStr = obj.dump();
        workerClients.at((int)temp_s)->publish(objStr);
        workerClients.at((int)temp_d)->publish(objStr);
```

Calling `dump()` again is redundant and consume more resources. We can reuse the same value.